src/org/zaproxy/zap/extension/ascanrulesBeta/InsecureHTTPMethod.java

```diff
@@ -62,7 +62,10 @@
 			{
 			"TRACE",
 			"TRACK",
-			"CONNECT"
+			"CONNECT",
+			"PUT",
+			"DELETE",
```

Comment:
The description of the the alert (and "extra info") should be updated to explain why these are insecure.